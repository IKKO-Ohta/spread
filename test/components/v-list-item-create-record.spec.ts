import Vue from 'vue'
import Vuetify from 'vuetify'
import { Wrapper, shallowMount } from '@vue/test-utils'
import VListItemCreateSheet from '@/components/v-list-item-create-sheet.vue'

Vue.use(Vuetify)
describe('VListItemCreateSheet', () => {
  test('snapshot', () => {
    const wrapper: Wrapper<VListItemCreateSheet> = shallowMount(VListItemCreateSheet)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
