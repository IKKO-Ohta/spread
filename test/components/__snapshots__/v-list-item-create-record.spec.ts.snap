// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VListItemCreateRecord snapshot 1`] = `
<section>
  <v-list-item-stub activeclass="" tag="div">
    <v-list-item-action-stub>
      <v-icon-stub>mdi-checkerboard-plus</v-icon-stub>
    </v-list-item-action-stub>
    <v-list-item-content-stub>
      <v-list-item-title-stub></v-list-item-title-stub>
    </v-list-item-content-stub>
  </v-list-item-stub>
  <v-dialog-stub opendelay="0" closedelay="0" contentclass="" maxwidth="600px" origin="center center" retainfocus="true" transition="dialog-transition" width="auto">
    <v-card-stub loaderheight="4" tag="div" class="form">
      <v-card-title-stub> create a new record</v-card-title-stub>
      <v-card-text-stub>
        <v-text-field-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" value="" backgroundcolor="" label="sheet name" loaderheight="2" clearicon="$clear" type="text" required=""></v-text-field-stub>
        <v-select-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" value="" appendicon="$dropdown" backgroundcolor="" label="Game" loaderheight="2" clearicon="$clear" type="text" valuecomparator="function deepEqual(a, b) {
  if (a === b) return true;

  if (a instanceof Date &amp;&amp; b instanceof Date) {
    // If the values are Date, they were convert to timestamp with getTime and compare it
    if (a.getTime() !== b.getTime()) return false;
  }

  if (a !== Object(a) || b !== Object(b)) {
    // If the values aren't objects, they were already checked for equality
    return false;
  }

  var props = Object.keys(a);

  if (props.length !== Object.keys(b).length) {
    // Different number of props, don't bother to check
    return false;
  }

  return props.every(function (p) {
    return deepEqual(a[p], b[p]);
  });
}" nodatatext="$vuetify.noDataText" items="Magic: the Gathering,Pokemon Card Game,Yu-Gi-Oh! Official Card Game,Weiss Schwaltz,Harth Stone,Shadowverse,Other BO1,Other BO3" itemcolor="primary" itemdisabled="disabled" itemtext="text" itemvalue="value" menuprops="[object Object]"></v-select-stub>
      </v-card-text-stub>
      <v-card-actions-stub>
        <v-spacer-stub></v-spacer-stub>
        <v-btn-stub tag="button" activeclass="" disabled="true" depressed="true" type="button">
          Create
        </v-btn-stub>
      </v-card-actions-stub>
    </v-card-stub>
  </v-dialog-stub>
</section>
`;

exports[`VListItemCreateSheet snapshot 1`] = `
<section>
  <v-list-item-stub activeclass="" tag="div">
    <v-list-item-action-stub>
      <v-icon-stub>mdi-checkerboard-plus</v-icon-stub>
    </v-list-item-action-stub>
    <v-list-item-content-stub>
      <v-list-item-title-stub></v-list-item-title-stub>
    </v-list-item-content-stub>
  </v-list-item-stub>
  <v-dialog-stub opendelay="0" closedelay="0" contentclass="" maxwidth="600px" origin="center center" retainfocus="true" transition="dialog-transition" width="auto">
    <v-card-stub loaderheight="4" tag="div" class="form">
      <v-card-title-stub>新しいシートを作成する</v-card-title-stub>
      <v-card-text-stub>
        <v-text-field-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" value="" backgroundcolor="" label="シートの名前" loaderheight="2" clearicon="$clear" type="text" required=""></v-text-field-stub>
        <v-select-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" value="" appendicon="$dropdown" backgroundcolor="" label="ゲームタイトル" loaderheight="2" clearicon="$clear" type="text" valuecomparator="function deepEqual(a, b) {
  if (a === b) return true;

  if (a instanceof Date &amp;&amp; b instanceof Date) {
    // If the values are Date, they were convert to timestamp with getTime and compare it
    if (a.getTime() !== b.getTime()) return false;
  }

  if (a !== Object(a) || b !== Object(b)) {
    // If the values aren't objects, they were already checked for equality
    return false;
  }

  var props = Object.keys(a);

  if (props.length !== Object.keys(b).length) {
    // Different number of props, don't bother to check
    return false;
  }

  return props.every(function (p) {
    return deepEqual(a[p], b[p]);
  });
}" nodatatext="$vuetify.noDataText" items="遊戯王 OCG,デュエル・マスターズ,マジック・ザ・ギャザリング,ポケモンカードゲーム,ヴァイスシュヴァルツ,カードファイト!! ヴァンガード,フューチャーカード バディファイト,Battle Spirits,Z/X,WIXOSS -ウィクロス-,BO1ゲーム,BO3ゲーム" itemcolor="primary" itemdisabled="disabled" itemtext="text" itemvalue="value" menuprops="[object Object]"></v-select-stub>
      </v-card-text-stub>
      <v-card-actions-stub>
        <v-spacer-stub></v-spacer-stub>
        <v-btn-stub tag="button" activeclass="" disabled="true" depressed="true" type="button">
          作成する
        </v-btn-stub>
      </v-card-actions-stub>
    </v-card-stub>
  </v-dialog-stub>
</section>
`;
